<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  
  
  
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
  <title>
    【Nature Cell Biology】消除碱基编辑诱导的全基因组和转录组脱靶突变 |
    
    月溪明</title>
  
  <link rel="shortcut icon" href="/images/dna.svg">
  
  
<link rel="stylesheet" href="/css/style.css">

  
  
<link rel="stylesheet" href="/fancybox/jquery.fancybox.min.css">

  
  
<script src="/js/pace.min.js"></script>

<meta name="generator" content="Hexo 5.4.0"></head>

<body>
  <main class="content">
    <section class="outer">
  <article id="post-文献导读12" class="article article-type-post" itemscope
  itemprop="blogPost" data-scroll-reveal>

  <div class="article-inner">
    
    <header class="article-header">
      

<h1 class="article-title" itemprop="name">
  【Nature Cell Biology】消除碱基编辑诱导的全基因组和转录组脱靶突变
</h1>



    </header>
    

    
    <div class="article-meta">
      <a href="/2021/06/30/%E6%96%87%E7%8C%AE%E5%AF%BC%E8%AF%BB12/" class="article-date">
  <time datetime="2021-06-30T08:50:13.000Z" itemprop="datePublished">2021-06-30</time>
</a>
      
    </div>
    

    
    
<div class="tocbot"></div>

    

    <div class="article-entry" itemprop="articleBody">
      
      
      
      <h1 id="Eliminating-base-editor-induced-genome-wide-and-transcriptome-wide-off-target-mutations"><a href="#Eliminating-base-editor-induced-genome-wide-and-transcriptome-wide-off-target-mutations" class="headerlink" title="Eliminating base-editor-induced genome-wide and transcriptome-wide off-target mutations"></a>Eliminating base-editor-induced genome-wide and transcriptome-wide off-target mutations</h1><span id="more"></span> 
<p>作者： 杨力  期刊：Nature Cell Biology  时间：2021.5</p>
<p><a target="_blank" rel="noopener" href="https://www.nature.com/articles/s41556-021-00671-4#Abs1">原文链接</a></p>
<h2 id="Abstract"><a href="#Abstract" class="headerlink" title="Abstract"></a>Abstract</h2><p>CRISPR-Cas9与胞苷脱氨酶（  cytidine  deaminases ）的融合导致了能够进行可编程C-to-T编辑的碱基编辑器( base  editors，BEs)，该编辑器具有临床应用潜力，但存在脱靶(off-target ，OT)突变。这里，作者使用一个可剪切的脱氧胞苷脱氨酶抑制剂(dCDI)结构域构建了一个 变形式BE (tBE)系统，该系统仅在全基因组和转录组范围的OT突变的背景水平上就能诱导高效的编辑。 产生后，tBE在OT位点与可裂解dCDI融合保持不活跃状态，因此消除了意外突变。 当结合在目标位点时，tBE被转化以切断dCDI结构域，并催化靶向脱氨基以进行精确的碱基编辑 。 通过双腺相关病毒(AAV)系统导入小鼠后，tBE系统在Pcsk9中创建了一个提前终止密码子，显著降低了血清中Pcsk9，导致总胆固醇下降~ 30-40%。  tBE的开发建立了一种高度特异性的碱基编辑系统，其体内疗效具有治疗应用的潜力。 </p>
<h2 id="Fig-1-identification-of-dCDi-domains"><a href="#Fig-1-identification-of-dCDi-domains" class="headerlink" title="Fig. 1 | identification of dCDi domains."></a>Fig. 1 | identification of dCDi domains.</h2><p><img src="/2021/06/30/%E6%96%87%E7%8C%AE%E5%AF%BC%E8%AF%BB12/1.png" alt></p>
<p>a.APOBEC  家族有两类，一类只有一个CDA 位点，另一类有两个CDA位点，其中一个位点具有催化活性，另一个位点没有催化活性但有调节功能。作者构建了10个BEs，分别包含和不包含inactive  but  regulatory  domain。</p>
<p>b c.包含inactive的CDA位点的APOBEC编辑效率显著降低。</p>
<p>def.为了验证一般性，作者将mA3,  hA3F  和 hA3B的inactive CDA位点连接到了mA3CDA1–BE3上，发现都有编辑效率降低。CDA2  of  mA3的抑制效果最强。</p>
<p>综上所述，这些调控域作为dCDIs对脱氧胞苷脱氨酶活性具有普遍的抑制作用。 </p>
<h2 id="Fig-2-Comparison-of-tBE-with-other-BEs"><a href="#Fig-2-Comparison-of-tBE-with-other-BEs" class="headerlink" title="Fig. 2 | Comparison of tBE with other BEs."></a>Fig. 2 | Comparison of tBE with other BEs.</h2><p><img src="/2021/06/30/%E6%96%87%E7%8C%AE%E5%AF%BC%E8%AF%BB12/2.png" alt></p>
<p>作者试图利用dCDI域开发一个BE系统来避免sgRNA不依赖的突变。理想情况下，由于dCDI的连接，这种BE不会引起可观察到的OT突变，但当与靶点结合时，可以通过切断dCDI而转化为活性;因此被称为tBE。 </p>
<p>作者构建了   tBE-V5–mA3 这一BE，并将其与其他的BE进行对比，发现 tBE-V5–mA3 上靶效果很好，包括 DNA甲基化水平高的位点和含GpC二核苷酸 的位点。并且在OTss sites 和 OT sites，只有背景水平的突变。</p>
<h2 id="Fig-3-optimization-of-hsgRNA-in-the-tBE-system"><a href="#Fig-3-optimization-of-hsgRNA-in-the-tBE-system" class="headerlink" title="Fig. 3 | optimization of hsgRNA in the tBE system."></a>Fig. 3 | optimization of hsgRNA in the tBE system.</h2><p><img src="/2021/06/30/%E6%96%87%E7%8C%AE%E5%AF%BC%E8%AF%BB12/3.png" alt></p>
<p>在比较编辑效率时，作者注意到sgRNA和hsgRNA之间的距离影响了tBE对目标的编辑效率。因此，作者测定了sgRNA与hsgRNA之间距离的影响(图3a、b)，发现当hsgRNA位于sgRNA的−91 bp至−34 bp(上游)区域时，tBE的编辑效率达到最佳（图3c）</p>
<p>由于hsgRNA在其结合位点也形成R-loop区域(图3d)，一些hsgRNA的靶位点偶尔会发生意外的C-to-T编辑 (图3e left)。 之前的研究表明使用带有截断间隔的guide RNA并不影响Cas9的结合 。 因此，作者研究了hsgRNA靶位点的非预期编辑是否也可以通过截断的hsgRNAs来抑制。 使用截断的hsgRNA时，hsgRNA靶区的非预期编辑减少  (Fig.  3e,f).  有10bp间隔的hsgRNA消除了意外的编辑 ，而目标编辑不受影响 。</p>
<h2 id="Fig-4-the-effect-of-split-TEV-protease-concentration-on-editing-efficiency-and-sgRNA-independent-OT-mutation"><a href="#Fig-4-the-effect-of-split-TEV-protease-concentration-on-editing-efficiency-and-sgRNA-independent-OT-mutation" class="headerlink" title="Fig. 4 | the effect of split-TEV protease concentration on editing efficiency and sgRNA-independent OT mutation."></a>Fig. 4 | the effect of split-TEV protease concentration on editing efficiency and sgRNA-independent OT mutation.</h2><p><img src="/2021/06/30/%E6%96%87%E7%8C%AE%E5%AF%BC%E8%AF%BB12/4.png" alt></p>
<p>作者还用不同数量的表达载体表达了分裂- tev系统(图4a)，发现在分裂- tev组分检测的蛋白水平上，目标编辑效率(图4b)和sgrna独立OTss位点的突变频率(图4c)均未受影响(图4d)。 </p>

      
    </div>
    <footer class="article-footer">
      <a data-url="https://mingzhentian.github.io/2021/06/30/%E6%96%87%E7%8C%AE%E5%AF%BC%E8%AF%BB12/" data-id="ckvtjevht000f0of9awuwaqu3" class="article-share-link">
        分享
      </a>
      
<ul class="article-tag-list" itemprop="keywords"><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/Bioinformatics/" rel="tag">Bioinformatics</a></li><li class="article-tag-list-item"><a class="article-tag-list-link" href="/tags/%E6%96%87%E7%8C%AE%E5%AF%BC%E8%AF%BB/" rel="tag">文献导读</a></li></ul>

    </footer>

  </div>

  
  
<nav class="article-nav">
  
  <a href="/2021/08/29/cs231-1/" class="article-nav-link">
    <strong class="article-nav-caption">前一篇</strong>
    <div class="article-nav-title">
      
      【cs231n】Lecture 1 - Overview 笔记
      
    </div>
  </a>
  
  
  <a href="/2021/06/30/cv/" class="article-nav-link">
    <strong class="article-nav-caption">后一篇</strong>
    <div class="article-nav-title">Curriculum Vitae</div>
  </a>
  
</nav>

  

  
  
  
  

</article>
</section>
    <footer class="footer">
  <div class="outer">
    <div class="float-right">
      <ul class="list-inline">
  
  <li><i class="fe fe-smile-alt"></i> <span id="busuanzi_value_site_uv"></span></li>
  
  <li><i class="fe fe-bookmark"></i> <span id="busuanzi_value_page_pv"></span></li>
  
</ul>
    </div>
    <ul class="list-inline">
      <li>月溪明 &copy; 2021</li>
      
      <li>Powered by <a href="http://hexo.io/" target="_blank">Hexo</a> <a href="https://github.com/MingzhenTian" target="_blank">My Github</a></li>
    </ul>
  </div>
</footer>
  </main>
  <aside class="sidebar">
    <button class="navbar-toggle"></button>
<nav class="navbar">
  
  <div class="logo">
    <a href="/"><img src="/images/dna.svg" alt="月溪明" style="width:3.5rem !important"></a>
  </div>
  
  <ul class="nav nav-main">
    
    <li class="nav-item">
      <a class="nav-item-link" href="/">主页</a>
    </li>
    
    <li class="nav-item">
      <a class="nav-item-link" href="/archives">时间轴</a>
    </li>
    
    <li class="nav-item">
      <a class="nav-item-link" href="/gallery">相册</a>
    </li>
    
    <li class="nav-item">
      <a class="nav-item-link" href="/tags">标签</a>
    </li>
    
    <li class="nav-item">
      <a class="nav-item-link" href="/about">关于我</a>
    </li>
    
    <li class="nav-item">
      <a class="nav-item-link nav-item-search" title="搜索">
        <i class="fe fe-search"></i>
        搜索
      </a>
    </li>
  </ul>
</nav>
<nav class="navbar navbar-bottom">
  <ul class="nav">
    <li class="nav-item">
      <div class="totop" id="totop">
  <i class="fe fe-rocket"></i>
</div>
    </li>
    <li class="nav-item">
      
      <a class="nav-item-link" target="_blank" href="/atom.xml" title="RSS Feed">
        <i class="fe fe-feed"></i>
      </a>
      
    </li>
  </ul>
</nav>
<div class="search-form-wrap">
  <div class="local-search local-search-plugin">
  <input type="search" id="local-search-input" class="local-search-input" placeholder="Search...">
  <div id="local-search-result" class="local-search-result"></div>
</div>
</div>
  </aside>
  
<script src="/js/jquery-2.0.3.min.js"></script>


<script src="/js/jquery.justifiedGallery.min.js"></script>


<script src="/js/lazyload.min.js"></script>


<script src="/js/busuanzi-2.3.pure.min.js"></script>



<script src="/fancybox/jquery.fancybox.min.js"></script>




<script src="/js/copybtn.js"></script>





<script src="/js/tocbot.min.js"></script>

<script>
  // Tocbot_v4.7.0  http://tscanlin.github.io/tocbot/
  tocbot.init({
    tocSelector: '.tocbot',
    contentSelector: '.article-entry',
    headingSelector: 'h1, h2, h3, h4, h5, h6',
    hasInnerContainers: true,
    scrollSmooth: true,
    positionFixedSelector: '.tocbot',
    positionFixedClass: 'is-position-fixed',
    fixedSidebarOffset: 'auto',
  });
</script>



<script src="/js/ocean.js"></script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        tex2jax: {
            inlineMath: [ ["$","$"], ["\\(","\\)"] ],
            skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code'],
            processEscapes: true
        }
    });
    MathJax.Hub.Queue(function() {
        var all = MathJax.Hub.getAllJax();
        for (var i = 0; i < all.length; ++i)
            all[i].SourceElement().parentNode.className += ' has-jax';
    });
</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML"></script>
</body>

</html>